docs/installation/instructions.rst

```diff
@@ -36,8 +53,7 @@ PyPI Installation
 
    .. note::
 
-       The exact path will depend on the major *and* minor Python version found by venv e.g.
-       /lib/python3.5/, /lib/python3.6/
+       The exact path will depend on the operating system, Python version, and other factors.
```

Comment:
Maybe link to https://raw.githubusercontent.com/pulp/pulp/3.0-dev/pulpcore/pulpcore/etc/pulp/server.yaml in case they want to put it in /etc/pulp/server.yaml and not want to go look for where venv installed pulp

Reply:
+1